tests/test_mf6_disu_bmi.py

```diff
@@ -1,112 +1,79 @@
 import numpy as np
+import pytest
 
 from xmipy import XmiWrapper
 
 
-def test_get_grid_face_count(flopy_disu, modflow_lib_path):
-    """Tests if the grid_face_count can be extracted"""
+@pytest.fixture
+def flopy_disu_mf6(flopy_disu, modflow_lib_path, request):
     mf6 = XmiWrapper(lib_path=modflow_lib_path, working_directory=flopy_disu.sim_path)
 
-    # Write output to screen:
+    # If initialized, call finalize() at end of use
+    request.addfinalizer(mf6.__del__)
+
+    # Write output to screen
     mf6.set_int("ISTDOUTTOFILE", 0)
 
-    try:
-        # Initialize
-        mf6.initialize()
+    return flopy_disu, mf6
 
-        prescribed_grid_face_count = flopy_disu.nrow * flopy_disu.ncol
 
-        # Getting the grid id from the model, requires specifying one variable
-        k11_tag = mf6.get_var_address("K11", flopy_disu.model_name, "NPF")
-        grid_id = mf6.get_var_grid(k11_tag)
-        actual_grid_face_count = mf6.get_grid_face_count(grid_id)
+def test_get_grid_face_count(flopy_disu_mf6):
+    """Tests if the grid_face_count can be extracted"""
+    mf6 = flopy_disu_mf6[1]
+    mf6.initialize()
 
-        assert prescribed_grid_face_count == actual_grid_face_count
-    finally:
-        mf6.finalize()
+    nrow = 3
+    ncol = 3
```

Comment:
Why are these values hardcoded now?

Reply:
I suppose to minimize complexity of the test, but referencing variables from the dataclass was also fine. I've restored it as before, but renaming "prescribed" to "expected".

Reply:
> I've restored it as before, but renaming "prescribed" to "expected".

Feedback from native speakers is always welcome :)